### ISSUE TITLE
Cannot compile

### ISSUE THREAD

**Issue by mlunax** [darktohka/audacious-plugin-rpc#6]
Every step like in README
```
❯ make
[ 50%] Linking CXX shared library libaudacious-plugin-rpc.so
/usr/bin/ld: ../libdiscord-rpc.a(discord_rpc.cpp.o): warning: relocation against `_ZSt7nothrow@@GLIBCXX_3.4' in read-only section `.text'
/usr/bin/ld: ../libdiscord-rpc.a(discord_rpc.cpp.o): relocation R_X86_64_PC32 against symbol `_ZSt7nothrow@@GLIBCXX_3.4' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: bad value
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/audacious-plugin-rpc.dir/build.make:104: libaudacious-plugin-rpc.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/audacious-plugin-rpc.dir/all] Error 2
make: *** [Makefile:149: all] Error 2
```
My versions:
```
g++ (GCC) 10.2.0
cmake version 3.19.3
discord-rpc: (commit)963aa9f3e5ce81a4682c6ca3d136cddda614db33
```

**Comment by Shaorrran** [comment]
@mlunax I have encountered the same the problem. The solution was to pass the `-DCMAKE_POSITION_INDEPENDENT_CODE=ON` flag to `cmake` when configuring the build for `discord-rpc`, like this
`cmake .. -DCMAKE_INSTALL_PREFIX=<installation path> -DCMAKE_POSITION_INDEPENDENT_CODE=ON`

and then proceed to build `discord-rpc` and then `audacious-plugin-rpc` as noted in respective READMEs.

Tested on `gcc (Debian 10.2.1-6) 10.2.1 20210110`, `cmake version 3.18.4` and `discord-rpc: (commit)963aa9f3e5ce81a4682c6ca3d136cddda614db33`

**Comment by darktohka** [comment]
Resolved by #8.
